apps/meteor/client/providers/VideoConfProvider.tsx

```diff
@@ -0,0 +1,92 @@
+import { IRoom } from '@rocket.chat/core-typings';
+import { useSetModal } from '@rocket.chat/ui-contexts';
+import React, { ReactElement, useState, ReactNode, useMemo, useEffect } from 'react';
+import { Unsubscribe } from 'use-subscription';
+
+import GenericModal from '../components/GenericModal';
+import { VideoConfContext, VideoConfPopupPayload } from '../contexts/VideoConfContext';
+import { VideoConfManager, DirectCallParams, ProviderCapabilities, CallPreferences } from '../lib/VideoConfManager';
+import VideoConfPopups from '../views/room/contextualBar/VideoConference/VideoConfPopups/VideoConfPopups';
+
+const VideoConfContextProvider = ({ children }: { children: ReactNode }): ReactElement => {
+	const [outgoing, setOutgoing] = useState<VideoConfPopupPayload | undefined>();
+	const setModal = useSetModal();
+
+	useEffect(
+		() =>
+			VideoConfManager.on('call/join', (props) => {
+				const open = (): void => {
+					const popup = window.open(props.url);
+
+					if (popup !== null) {
+						return;
+					}
+
+					setModal(
+						<GenericModal
+							variant='warning'
+							title='Action blocked'
+							confirmText={'Open_again'}
+							onConfirm={open}
+							onCancel={(): void => setModal()}
+							onClose={(): void => setModal()}
+						>{`Your browser has blocked the page {PAGE}. <br/> Consider enable popups for this domain url`}</GenericModal>,
```

Comment:
The content of this modal is still pending

Reply:
@pierre-lehnen-rc fixed!